src/main/resources/messages/LibertyBundles.properties

```diff
@@ -130,7 +130,7 @@ run.config.liberty.project=Liberty project
 run.config.liberty.project.tool.tip=Select the build file for your Liberty project.
 run.config.start.parameters=Start Parameters
 run.config.start.parameters.tool.tip=eg. -DhotTests=true (Maven) --hotTests=true (Gradle)
-specify.custom.parameters.for.the.mvn.liberty.dev.command=Specify custom parameters for the mvn liberty:dev command.
+specify.custom.parameters.for.the.liberty.dev.command=Specify custom parameters for Liberty Start command.
```

Comment:
for consistency wit the user guide, recommend using "Liberty Start action", particularly if the custom parameters are applied when user selects that action from IDE menu

Reply:
@dmuelle So can we update the message to `Specify custom parameters for Liberty Start action` ?

Reply:
Hi @anusreelakshmi934 , @dmuelle and I further discussed this scenario and we landed on a new idea.
We would like to use the message `Specify custom parameters for Liberty dev mode.` However, we would like to have that message as a placeholder inside the input box (i.e. is greyed out and disappears once a user starts typing). I am not sure if that is possible with these UI elements.

We would also like to change `Start Parameters:` to `Start parameters:`.